Tests/NIOTests/EventLoopTest.swift

```diff
@@ -228,4 +228,24 @@ public class EventLoopTest : XCTestCase {
             XCTAssertNoThrow(try group.syncShutdownGracefully())
         #endif
     }
+    
+    public func testCurrentEventLoop() throws {
+        let group = MultiThreadedEventLoopGroup(numThreads: 2)
+        
+        let loop1 = group.next()
+        let currentLoop1 = try loop1.submit {
+            return MultiThreadedEventLoopGroup.currentEventLoop
+        }.wait()
+        XCTAssertTrue(loop1 === currentLoop1)
+        
+        let loop2 = group.next()
+        let currentLoop2 = try loop2.submit {
+            return MultiThreadedEventLoopGroup.currentEventLoop
+        }.wait()
+        XCTAssertTrue(loop2 === currentLoop2)
+        XCTAssertFalse(loop1 === loop2)
+        
+        XCTAssertNil(MultiThreadedEventLoopGroup.currentEventLoop)
+        XCTAssertNoThrow(try group.syncShutdownGracefully())
```

Comment:
Can you add another test that grabs a weak ref to the event loop and confirm that it doesn't leak?

Reply:
Good idea... will do.